### ISSUE TITLE
Default branch not syncing

### ISSUE THREAD

**Issue by ricardoamaro** [lemurheavy/coveralls-public#1589]
On Github, we changed the default branch from main to master. After a few days and clicking the sync button, the default branch in  Coveralls repo is still showing main.

![image](https://user-images.githubusercontent.com/384962/137918130-591da324-958a-43fa-ae77-c715ad25a122.png)

Since clicking "Sync repos" does not update the default branch how can we fix this issue on https://coveralls.io/github/acquia/epe-aop-work-order-factory ?

**Comment by afinetooth** [comment]
Hi @ricardoamaro,

Sorry for the trouble. I believe the issue was with the Github access token of the current repo owner (or its privilege level). To test, I made you the owner (since I saw that you had Admin access) and retried the SYNC button in settings:
https://coveralls.io/github/acquia/epe-aop-work-order-factory/settings

![Screen Shot 2021-10-20 at 1 49 47 PM](https://user-images.githubusercontent.com/918667/138170041-dd67b31c-d561-4511-a29f-f22ec4d62fa0.png)

Whereas, previously, the response from Github was "No new info," syncing as you returned "Info updated!" and your default branch became **master**:

![Screen Shot 2021-10-20 at 1 52 11 PM](https://user-images.githubusercontent.com/918667/138170328-e3af13cf-c994-419a-994d-b84c1d5e14b7.png)

Worth noting for other users with this issue, you'll want to:

- Make sure your repo owner has Admin access to the repo at Github
- Give that user Admin access at Github if it doesn't
- If not possible, email us at support@coveralls.io and we'll change the owner for you.
